src/steamship/agents/basic_chat/basic_chat_agent.py

```diff
@@ -0,0 +1,94 @@
+from typing import List
+
+from steamship import Block, SteamshipError
+from steamship.agents.basic_chat.basic_chat_output_parser import BasicChatOutputParser
+from steamship.agents.schema import Action, AgentContext, ChatAgent, ChatLLM
+from steamship.data.tags.tag_constants import RoleTag
+
+DEFAULT_PROMPT = """You are a helpful AI assistant.
+
+You chat with users are eager to engage on a variety of topics. You answer their questions, provide help thinking
+through challenges they have, and engage as a trusted assistant."""
+
+
+class BasicChatAgent(ChatAgent):
+    """BasicChatAgent implements a conversational agent with no function calling or tool reasoning.
```

Comment:
I think this documentation is misleading and will get us into trouble.  `ChatAgent`s in our codebase are for selection actions. They are not for carrying on conversations (directly).

I fear that if what you want is *direct* LLM responses, trying to bend `AgentService` to fit that need _may_ be problematic.

Should we have a `ConversationService` as a parallel concept?  Something that doesn't have the indirection of `run_agent` and `run_action` just to issue a call to an LLM and stream back a response?

Reply:
I think the `ConversationService` route is a perfectly fine idea, but since the rest of our codebase (e.g. transports -- which the UI is also built around) depends upon AgentService so it would involve a larger rewrite.

I don't understand the concern about an Agent that always selects the `FinishAction`. It seems to me that an agent which behaves more narrowly is no different than an agent who behaves more broadly.

The current description of agent in our codebase is:

    """Agent is responsible for choosing the next action to take for an AgentService.

    It uses the provided context, and a set of Tools, to decide on an action that will
    be executed by the AgentService.
    """

If we just add the word `possibly` before `a set of Tools` then I'm pretty an agent that chit chats fits that description perfectly fine.